Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -5644,11 +5645,6 @@ bool LocalFrameView::removeScrollableArea(ScrollableArea* scrollableArea)
     return false;
 }
 
-bool LocalFrameView::containsScrollableArea(ScrollableArea* scrollableArea) const
```

Comment:
I also think it's fine to leave this in. Even if it's not used now, we might want to use it in future.

Reply:
Should I do early return in these to make them future proof as well to handle 'null' case?